docs/release-notes/dictionary-releases.md

```diff
@@ -19,6 +19,38 @@ Coming soon.
 None to report.
 ------>
 
+## Release 1.8
+
+**Release Date:** April 27, 2021
+
+### Updates
+Two **new** data type tables have been added to the dictionary for submission:
+1. **Immunotherapy**: For treatment regimens that include immunotherapy, these details can now be submitted.  Similar to the `chemotherapy` and `hormone_therapy`, `immunotherapy drugs are recorded using the standardized vocabulary from the [RxNorm database](https://www.nlm.nih.gov/research/umls/rxnorm), provided by the NIH. 
+
+2. **Family History**: A donor's familial relationships and familial cancer history. 
+
+The following updates are also included:
+- Validation has been added so that when the `treatment_type = No Treatment`, non-applicable core fields do not need to be submitted. 
+
+
+
+### Bug Fixes
+- Previously missing stage groups `Stage IVE` and `Stage IVS` for Ann Abor staging system has been added, as well as adding support for the `Stage II bulky` for the Lugano staging system. 
+
+- In the `primary_diagnosis` table, there was a misleading combination of field indicating the status/number of lymph nodes examined during clinical diagnosis. When the `number_lymph_nodes_examined` is left blank, a value of 0 for `number_lymph_nodes_positive` could be unclearly misinterpreted as several things: 
+  - an unknown number of lymph nodes were tested or
+  - none were found to be positive for cancer or 
+  - no lymph nodes were examined.
+
+ To make this more accurate to clinical use cases, a new field `lymph_nodes_examined_status` has been added to indicate  if lymph nodes were examined.  Depending on the status, the required `number_of_lymph_nodes_examined` and `number_of_lymph_nodes_positive` can now accurately reflect clinical use cases. For example: 
+
+|lymph_nodes_examined_status|number_lymph_nodes_examined|number_lymph_nodes_positive|
+|----|-----|-----|
+|Yes|Optional: If left blank, then means unknown/missing. If number is submitted, it must be greater than 0|Required: If number_lymph_nodes_examined is > 0, then number_lymph_nodes_positive >= number_lymph_nodes_examined |
```

Comment:
suggestion for reword
Optional: If left blank, `this` means unknown/missing. If number is submitted, it must be greater than 0